styles/tokens/input.json

```diff
@@ -132,29 +123,104 @@
+        "value": "2px",
+        "attributes": {
+          "category": "size"
+        },
+        "file": "settings",
+        "themable": true
+      },
+      "box-shadow": {
+        "comment": "The box shadow on Inputs when focused.",
+        "value": "0 0 0 2px {purple.value}, 0 2px 8px {black-tint.30.value}",
+        "attributes": {
+          "category": "content"
+        },
+        "file": "settings",
+        "themable": true
+      }
     },
-    "disabled-border-color": {
-      "comment": "The border color on disabled Inputs.",
-      "value": "{black-tint.50.value}",
-      "file": "settings",
-      "themable": true
+    "error": {
+      "box-shadow": {
+        "comment": "The box shadow on Inputs there is an error.",
+        "value": "0 0 0 1px {orange.value}",
+        "attributes": {
+          "category": "content"
+        },
+        "file": "settings",
+        "themable": true
+      },
+      "focus-box-shadow": {
+        "comment": "The box shadow on Inputs there is an error and it's focused.",
+        "value": "0 0 0 1px {orange.value}, 0 2px 8px {black-tint.30.value}",
+        "attributes": {
+          "category": "content"
+        },
+        "file": "settings",
+        "themable": true
+      },
+      "border-width": {
+        "comment": "The border width on Inputs when there is an error.",
+        "value": "1px",
+        "attributes": {
+          "category": "size"
+        },
+        "file": "settings",
+        "themable": true
+      },
+      "border-color": {
+        "comment": "The border color on Inputs when there is an error.",
+        "value": "{orange.value}",
+        "file": "settings",
+        "themable": true
+      }
     },
-    "disabled-color": {
-      "comment": "The color on disabled Inputs.",
-      "value": "{black-tint.50.value}",
-      "file": "settings",
-      "themable": true
+    "disabled": {
+      "border-color": {
+        "comment": "The border color on disabled Inputs.",
+        "value": "{black-tint.50.value}",
+        "file": "settings",
+        "themable": true
+      },
+      "border-width": {
+        "comment": "The border width on disabled Inputs.",
+        "value": "0",
+        "attributes": {
+          "category": "size"
+        },
+        "file": "settings",
+        "themable": true
+      },
+      "color": {
+        "comment": "The color on disabled Inputs.",
+        "value": "#808080",
+        "file": "settings",
+        "themable": true
+      },
+      "background-color": {
+        "comment": "The background color on disabled Inputs.",
+        "value": "{black-tint.15.value}",
+        "file": "settings",
+        "themable": true
+      },
+      "box-shadow": {
+        "comment": "The box shadow on disabled Inputs.",
+        "value": "none",
+        "attributes": {
+          "category": "content"
+        },
+        "file": "settings",
+        "themable": true
+      }
     },
     "readonly-border-color": {
```

Comment:
Let's take a look at this -- we got rid of all borders on inputs so this might not be necessary